lib/tasks/commit_backfill.rake

```diff
@@ -63,7 +56,7 @@ namespace :spree_avatax do
         detaillevel: 'Tax',
       }
 
-      history_result = tax_svc.gettaxhistory(history_request)
+      history_result = SpreeAvatax::Shared.tax_svc.gettaxhistory(history_request)
```

Comment:
Yeah, I think it would be good to actually keep `tax_svc` public as an easy way for client to hit Avatax endpoints we haven't used/tested/etc.